lego-webapp/cypress/support/utils.ts

```diff
@@ -105,16 +105,15 @@ export const setDatePickerDate = (
   field(name).click();
 
   if (isNextMonth) {
-    cy.get('ion-icon[name="arrow-forward-outline"]')
-      .first()
-      .should('not.be.disabled')
+    cy.get(`button:not(:disabled)${c('prevOrNextMonth')}`)
+      .contains(new RegExp('^' + date + '$', 'g'))
+      .click();
+  } else {
+    cy.get(`button:not(:disabled):not(${c('prevOrNextMonth')})`)
+      .contains(new RegExp('^' + date + '$', 'g'))
       .click();
   }
```

Comment:
Might i suggest:

`cy.get(`button:not(:disabled)${isNextMonth ? c('prevOrNextMonth') : `:not(${(c('prevOrNextMonth')}})` `)
      .contains(new RegExp('^' + date + '$', 'g'))
      .click();
`
🤣